mrp_bom_component_menu/readme/USAGE.md

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+1.  Go to menu *Manufacturing -\> Master Data -\> Bill of Material
+    Components*.
```

Comment:
```suggestion
1.  Go to menu *Manufacturing -\> Products -\> Bill of Material
    Components*.
```
I see that the Menu name got changed from previous versions. We can update the README to make it better understandable.